Pal/src/host/Linux-SGX/sgx_main.c

```diff
@@ -644,6 +646,7 @@ int load_manifest (int fd, struct config_store ** config_ptr)
 
     if (ret < 0) {
         SGX_DBG(DBG_E, "can't read manifest: %s\n", errstring);
+        free(config);
```

Comment:
It's also better to munmap "config_raw" as well.

Reply:
Sure, I will do that, thanks,